.github/workflows/release-spec-prod.yml

```diff
@@ -0,0 +1,23 @@
+name: '(PROD) Release OpenAPI Spec'
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 9 * * *' # Run at 9AM UTC
+permissions:
+  contents: write
+jobs:
+  run-validations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      - name: Setup Node.js
+        uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b
+        with:
+          node-version: 20
+      - name: Install Spectral
+        run: npm install -g @stoplight/spectral-cli@6.11.1 && spectral --version
```

Comment:
[nit] should we make the version a variable? is it even possible? sorry I'm learning XD

Reply:
yes, I will do it in a follow up PR 👍

Reply:
Made comment above that will enable dependabot support

Reply:
https://github.com/mongodb/openapi/pull/97

Reply:
based on this https://github.com/mongodb/openapi/pull/96#discussion_r1687833169 we won't need the install step